2024-05-30 15:51:52|   DEBUG|testAction.py[:57]|ffff
2024-05-30 15:51:52|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:52|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:52|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:52|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:52|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:52|   DEBUG|testAction.py[:57]|hhhh
2024-05-30 15:51:52|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:52|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:52|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:52|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:52|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:52|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:52|   DEBUG|testAction.py[:57]|gggg
2024-05-30 15:51:52|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:52|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:52|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:52|   DEBUG|testAction.py[:57]|iiii
2024-05-30 15:51:52|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:52|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:52|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:52|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:52|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:52|   DEBUG|testAction.py[:57]|jjjj
2024-05-30 15:51:52|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:52|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:52|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:52|   DEBUG|testAction.py[:57]|kkkk
2024-05-30 15:51:52|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:52|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:52|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:52|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:52|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:52|   DEBUG|testAction.py[:57]|llll
2024-05-30 15:51:52|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:53|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:53|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:53|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:53|   DEBUG|testAction.py[:57]|oooo
2024-05-30 15:51:53|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:53|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:53|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:53|   DEBUG|testAction.py[:57]|pppp
2024-05-30 15:51:53|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:51:53|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:53|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:53|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:53|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:51:53|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:51:53|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:51:53|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 15:54:36|   DEBUG|testAction.py[:57]|admin
2024-05-30 15:54:36|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:36|   DEBUG|testAction.py[:57]|sniper
2024-05-30 15:54:36|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:36|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:36|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:36|   DEBUG|testAction.py[:57]|sniper
2024-05-30 15:54:36|   DEBUG|testAction.py[:58]|111111
2024-05-30 15:54:36|   ERROR|testAction.py[:73]|用户名或密码错误
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 67, in dologin
    raise RuntimeError("用户名或密码错误")
RuntimeError: 用户名或密码错误
2024-05-30 15:54:36|    INFO|testAction.py[:78]|response json: {'code': 11, 'msg': '用户名或密码错误'}
2024-05-30 15:54:36|   DEBUG|testAction.py[:57]|apple
2024-05-30 15:54:36|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:36|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:36|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:36|   DEBUG|testAction.py[:57]|aaaa
2024-05-30 15:54:36|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:36|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:36|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:36|   DEBUG|testAction.py[:57]|bbbb
2024-05-30 15:54:36|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|cccc
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|dddd
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|eeee
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|ffff
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|hhhh
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|gggg
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|iiii
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|jjjj
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:37|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:37|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:37|   DEBUG|testAction.py[:57]|kkkk
2024-05-30 15:54:37|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:38|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:38|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:38|   DEBUG|testAction.py[:57]|llll
2024-05-30 15:54:38|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:38|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:38|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:38|   DEBUG|testAction.py[:57]|oooo
2024-05-30 15:54:38|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:38|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:38|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:38|   DEBUG|testAction.py[:57]|pppp
2024-05-30 15:54:38|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:54:38|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:38|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:54:38|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:54:38|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:57:48|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:57:51|   DEBUG|testAction.py[:218]|1
2024-05-30 15:57:51|   DEBUG|testAction.py[:219]|10
2024-05-30 15:57:51|   DEBUG|testAction.py[:223]|0
2024-05-30 15:57:51|   DEBUG|testAction.py[:224]|10
2024-05-30 15:58:01|   DEBUG|testAction.py[:218]|19
2024-05-30 15:58:01|   DEBUG|testAction.py[:219]|10
2024-05-30 15:58:01|   DEBUG|testAction.py[:223]|180
2024-05-30 15:58:01|   DEBUG|testAction.py[:224]|190
2024-05-30 15:58:11|   DEBUG|testAction.py[:218]|18
2024-05-30 15:58:11|   DEBUG|testAction.py[:219]|10
2024-05-30 15:58:11|   DEBUG|testAction.py[:223]|170
2024-05-30 15:58:11|   DEBUG|testAction.py[:224]|180
2024-05-30 15:58:12|   DEBUG|testAction.py[:218]|17
2024-05-30 15:58:12|   DEBUG|testAction.py[:219]|10
2024-05-30 15:58:12|   DEBUG|testAction.py[:223]|160
2024-05-30 15:58:12|   DEBUG|testAction.py[:224]|170
2024-05-30 15:58:39|   DEBUG|testAction.py[:57]|admin
2024-05-30 15:58:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:39|   DEBUG|testAction.py[:57]|sniper
2024-05-30 15:58:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:39|   DEBUG|testAction.py[:57]|sniper
2024-05-30 15:58:39|   DEBUG|testAction.py[:58]|111111
2024-05-30 15:58:39|   ERROR|testAction.py[:73]|用户名或密码错误
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 67, in dologin
    raise RuntimeError("用户名或密码错误")
RuntimeError: 用户名或密码错误
2024-05-30 15:58:39|    INFO|testAction.py[:78]|response json: {'code': 11, 'msg': '用户名或密码错误'}
2024-05-30 15:58:39|   DEBUG|testAction.py[:57]|apple
2024-05-30 15:58:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:39|   DEBUG|testAction.py[:57]|aaaa
2024-05-30 15:58:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|bbbb
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|cccc
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|dddd
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|eeee
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|hhhh
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|ffff
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|gggg
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|iiii
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:40|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:40|   DEBUG|testAction.py[:57]|jjjj
2024-05-30 15:58:40|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:41|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:41|   DEBUG|testAction.py[:57]|kkkk
2024-05-30 15:58:41|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:41|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:41|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:41|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:41|   DEBUG|testAction.py[:57]|llll
2024-05-30 15:58:41|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:41|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:41|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:41|   DEBUG|testAction.py[:57]|oooo
2024-05-30 15:58:41|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:41|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:41|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:41|   DEBUG|testAction.py[:57]|pppp
2024-05-30 15:58:41|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:58:41|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:41|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:58:41|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:58:41|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 16:11:38|   DEBUG|testAction.py[:57]|admin
2024-05-30 16:11:38|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|sniper
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|sniper
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|111111
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 16:11:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 16:11:39|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 16:11:39|   ERROR|testAction.py[:73]|用户名或密码错误
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 67, in dologin
    raise RuntimeError("用户名或密码错误")
RuntimeError: 用户名或密码错误
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 11, 'msg': '用户名或密码错误'}
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|apple
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 16:11:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 16:11:39|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|aaaa
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 16:11:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 16:11:39|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|bbbb
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 16:11:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 16:11:39|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|cccc
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 16:11:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 16:11:39|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|dddd
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 16:11:39|   DEBUG|testAction.py[:57]|eeee
2024-05-30 16:11:39|   DEBUG|testAction.py[:58]|123456
2024-05-30 16:11:39|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 16:11:39|   ERROR|testAction.py[:148]|日期不在允许范围内
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 131, in orderAddCommit
    raise Exception("日期不在允许范围内")
Exception: 日期不在允许范围内
2024-05-30 16:11:39|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 16:11:39|    INFO|testAction.py[:105]|载入主页面...
